src/nrnpython/CMakeLists.txt

```diff
@@ -87,33 +87,87 @@ else()
   set(BUILD_MEMACS_TRUE "#")
 endif()
 
-# =============================================================================
-# All variables are set, prepare setup.py for python module install
-# =============================================================================
-nrn_configure_file(setup.py src/nrnpython)
 
 # =============================================================================
-# Setup MinGW toolchain for setuptools
+# nrnpython libraries
 # =============================================================================
-if(MINGW)
-  file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/setup.cfg "[build]\ncompiler=mingw32")
-endif()
+add_library(rxdmath SHARED ${CMAKE_CURRENT_SOURCE_DIR}/rxdmath.c)
+install(TARGETS rxdmath DESTINATION ${NRN_INSTALL_SHARE_LIB_DIR})
 
-# =============================================================================
```

Comment:
just a detail, but i suppose you'd want to keep this title comment for the above two declarations since that is rxdmath not nrnpython